From b0fe2b087923abc9bf8adc1e43d933676057752f Mon Sep 17 00:00:00 2001
From: Ken Lierman <ken.lierman@windriver.com>
Date: Tue, 12 Mar 2013 11:39:31 -0700
Subject: Rename su, use the one from the superuser apk instead

see packages/apps/Superuser for details.

Change-Id: Ic5fa138ce332067dd72f774fc54bf913e54540b5
Depends-Change-Id: Id7eba74ff81c0f12a8cea46447e88d89d2bf28d5
For: AXIA-2163
---
 su/Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/su/Android.mk b/su/Android.mk
index 0593cc9..e1fe524 100644
--- a/su/Android.mk
+++ b/su/Android.mk
@@ -3,7 +3,7 @@ include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES:= su.c
 
-LOCAL_MODULE:= su
+LOCAL_MODULE:= su-old
 
 LOCAL_FORCE_STATIC_EXECUTABLE := true
 
-- 
1.7.4.1

